model/shared/src/main/scala/explore/model/ElevationPlotOptions.scala

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model
+
+import cats._
+import explore.model.enum.PlotRange
+import explore.model.enum.TimeDisplay
+import lucuma.core.enum.Site
+import monocle.Focus
+
+final case class ElevationPlotOptions(
+  site:  Site,
+  range: PlotRange,
+  time:  TimeDisplay
+)
+
+object ElevationPlotOptions {
+  val site  = Focus[ElevationPlotOptions](_.site)
+  val range = Focus[ElevationPlotOptions](_.range)
+  val time  = Focus[ElevationPlotOptions](_.time)
+
+  val Default =
+    ElevationPlotOptions(Site.GS, PlotRange.Night, TimeDisplay.Site)
+
+  implicit val targetVisualOptionsEq: Eq[ElevationPlotOptions] =
```

Comment:
Someday names won't be necessary. But for now this should probably match the class. 😄

Reply:
Some day as in scala 3 auto derivation?

Reply:
Well, that too. But I don't think `given`s have to have names in scala 3